src/freenas/usr/local/lib/middlewared_truenas/plugins/enclosure_/map.py

```diff
@@ -40,6 +40,17 @@
             MappingSlot(0, 2, False),
         ]),
     ]),
+    ProductMapping(re.compile(r"(TRUE|FREE)NAS-MINI-3.0-X$"), [
```

Comment:
The `ProductMapping` section of this file seems to have many duplicated entries.